### ISSUE TITLE
MARI run summing in re-runs

### ISSUE THREAD

**Issue by ElliotAOram** [ISISScientificComputing/autoreduce#392]
Issue raised by: [user]

### What?
The sum runs option in MARI (on resubmit in the webapp) does not work. Instead of submitting a range of files as one parameter, it sumbits them individually 
* The reduction algorithm uses all the data files in the range for *ONE* reduction
* A single data file is produced
* The data file is named correctly (`<first_run>_to_<last_run>`) 

### Where?
[see here](https://github.com/ISISScientificComputing/autoreduce/blob/master/WebApp/autoreduce_webapp/reduction_variables/views.py#L469). 
This will be handled in the message construction on the WebApp. The message must be constructed in such a way that the Queue can understand that it is to sum several data files together.

### How?
User request

### How to test the issue is resolved
* Go to the resubmit run page for MARI
* Enter a range of runs
* Check the `sum_runs` option. 
* Click resubmit
* Check that a single file is produced in ceph with the correct naming convention
* @ElliotAOram has test output files these can be validated against

**Comment by ElliotAOram** [comment]
The work done so far allows sum runs to be detected by the backend and refactors a large amount of the code to enable an easy way to submit runs from a range individually vs. a summed range. 

Unfortunately, this reaches a dead end as there would have to be a major rework of how the database schema works to enable any further progression. Given the lack of testing with in the webapp I suggest this issue goes on hold for now as it is incredibly high risk.

**Comment by ElliotAOram** [comment]
Adding a summed option to the schema (which states the last run number in the sequence) seems logical. I may experiment with this as an additive approach could be lower risk (assuming the field is set so that it can be NULL)

**Comment by ElliotAOram** [comment]
This change will not yield the results we want. In order to properly implement this (rather than a quick fix that will cause additional problems down the line). It will require a large change to the database schema. A brief think about this suggests:
* `Run` needs to be it's own table, with a linker table to `ReductionJob` (Previously `ReductionRun`)
* `DataLocations` requires a linker table to `ReductionJob`
Both of these changes are to enable many to many relationships between physical runs from ISIS and reduction jobs

**Comment by ElliotAOram** [comment]
Given the complexity of these changes, this will be put on hold until this requirement can be validated with SANS / Reflectometry who also require this. It is important that we implement a solution that will work across all instruments / reductions.

**Comment by ElliotAOram** [comment]
Finally, as a temporary measure, all run summation options will be removed from reduction parameters

**Comment by ElliotAOram** [comment]
blocked by database redesign

**Comment by dtasev** [comment]
We need to discuss this. DB redesign is probably not happening - this should be handled by MARI reduce scripts. Reflectometry and SANS seem to be able to do this with writing out extra metadata.

Guess we'll come back to this at some point, but for now it's getting closed
